codemp/client/cl_main.cpp

```diff
@@ -2832,6 +2859,8 @@ void CL_Init( void ) {
 	cl_downloadPrompt = Cvar_Get( "cl_downloadPrompt", "1", CVAR_ARCHIVE, "Confirm pk3 downloads from the server" );
 	cl_downloadOverlay = Cvar_Get( "cl_downloadOverlay", "1", CVAR_ARCHIVE, "Draw download info overlay" );
 
+	cl_downloadProtocol = Cvar_Get("cl_downloadProtocol", "", CVAR_INTERNAL, "Select the download protocol to use");
```

Comment:
Should probably set a default? 'http'

Reply:
If there is no download protocol available, it should be blank, as far as I know. Relevant check:

https://github.com/JakeFTL/Mystic-Lugormod/blob/93b6b0fed6406d271e84be32545e371c399acccc/codemp/client/cl_main.cpp#L1410-L1415